app/dashboard/views.py

```diff
@@ -2808,3 +2817,29 @@ def get_hackathons(request):
         'hackathons': events,
     }
     return TemplateResponse(request, 'dashboard/hackathons.html', params)
+
+@require_POST
+@login_required
+def upload_profile_banner(request):
```

Comment:
Since we aren't actually uploading a file, can we change the name of this function as well as the comment to reflect what this function actually does?